bin/rose-test-battery

```diff
@@ -40,6 +40,7 @@
 . $(dirname $0)/../lib/bash/rose_init
 . $(dirname $0)/../lib/bash/rose_log
 rose_init
+mkdir -p ~/.metomi
```

Comment:
There can be race condition for the creation of `~/.metomi/` when we run the test batteries in parallel.